src/components/eip3643/admin/CreateIdentityFactory.tsx

```diff
@@ -87,6 +94,16 @@ export default function CreateIdentityFactory() {
           >
             Create identity {!!currentIdentityAddress && "[Already created]"}
           </Button>
+          {hederaAccountIdError && (
+            <Alert status="error" mt="4">
+              <AlertIcon />
+              <AlertTitle>Hedera Account Id conversion error!</AlertTitle>
+              <AlertDescription>
+                Hedera Account Id detected. But here is an error converting it
+                to EVM address.
+              </AlertDescription>
+            </Alert>
+          )}
```

Comment:
we can consolidate and reuse this error msg, considering we might have more places to use it in the future